### ISSUE TITLE
Get HAR from Firefox on Android

### ISSUE THREAD

**Issue by soulgalore** [devtools-html/har-export-trigger#42]
Hi @janodvarko since a while ago Firefox can be used through Geckodriver on a Android phone. Installing HAR-export-trigger works fine but getting the HAR doesn't work. I guess the problem is that devtools and that the network tab needs to be open. I wanted to check if something changed the last year in the Firefox, hoping that maybe there could be a fix nowadays to avoid having devtools open?

Best
Peter

**Comment by janodvarko** [comment]
Hey Peter!
The Network panel doesn't have to be activated, but the DevTools Toolbox still needs to be opened to start up the backend for intercepting HTTP.

Honza

**Comment by soulgalore** [comment]
Hi Jan, ah ok, I was running 0.6.0 but upgraded to 0.6.1 and I guess then that fixes the problem, let me try.

Thanks!

**Comment by soulgalore** [comment]
Hi again @janodvarko so I upgraded to 0.6.1 and tested on my phone but didn't get it to work, and I guess it is because there's no devtools toolbox on Android? Do you know if someone tried running it on a phone?

Best
Peter

**Comment by janodvarko** [comment]
The toolbox is not available on Android (devtools/client is not bundled on those devices), you'd have to use remote debugging to record HAR. But since you mention Geckodriver, I guess the idea is to use this from automation?

**Comment by soulgalore** [comment]
Yes exactly. Since latest release of Geckodriver it works fine to drive Firefox on your Android phone. We got it up and running now and last piece for me is to get a HAR :) We kind have parts of the information from the geckoProfiler but it would be great if it would work with the plugin.
